server/routes/recipeRoute.js

```diff
@@ -13,6 +13,10 @@ const recipeRoute = (app) => {
     recipeController.getAllRecipes);
   app.post('/api/v1/recipes/:recipeId(\\d+)/votes/', authMiddleware, recipeExist,
     recipeController.voteRecipe);
+  app.post('/api/v1/users/:userId(\\d+)/recipes/:recipeId(\\d+)/', authMiddleware, recipeExist,
```

Comment:
why do you have to use this (\\d+) in your API?

Reply:
This is to check that its only numeric data type that is passed in.